tensorflow/python/platform/gfile.py

```diff
@@ -382,13 +382,15 @@ def ListDirectory(directory, return_dotfiles=False):  # pylint: disable=invalid-
   return files
 
 
-def Walk(top, topdown=1, onerror=None):
+def Walk(top, topdown=1, onerror=None, followlinks=False):
```

Comment:
I don't think there are very many uses of Walk right now, and it might make more sense to make followlinks the default, so we don't have to pass that everywhere.  Maybe just change line 404 to add followlinks=True and nothing else is needed?